templates/definition/charger/tplink.yaml

```diff
@@ -12,5 +12,5 @@ params:
     default: 15
 render: |
   type: tplink
-  uri: {{ .host }}
+  uri: {{ .host }}  # tplink device ip address (local)
```

Comment:
Die Kommentare hier landen nicht in der Doku. Der render Code wird nur evcc-intern verwendet. Du kannst den Text aber als `hint` and den jeweiligen Parameter schreiben. Dann wird er in `docs` als Kommentar und in `configure` als Hilfetext angezeigt.

Siehe:
https://github.com/evcc-io/evcc/blob/master/templates/definition/charger/fritzdect.yaml#L19-L21